server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -45,15 +45,15 @@ const AssessmentDayDataController = {
           .catch((err) => console.error(err))
 
         if (subject_assessments.length) {
-          AssessmentDayDataModel.saveMany(
+          AssessmentDayDataModel.createMany(
             dataDb,
             collection,
             subject_assessments
           )
-            .then(() => console.log('inserting new data'))
-            .catch((error) => {
-              throw new Error(error)
-            })
+          .then(() => console.log('inserting new data'))
+          .catch((error) => {
+            throw new Error(error)
+          })
```

Comment:
do we need this? Isn't this the same as not catching the error?

Reply:
Yup, it is. It's not really relevant to the ticket but I'll do a quick cleanup, won't take but a minute.

Reply:
d0a95d6